### PR TITLE
LES Turbulence unit tests

### PR DESCRIPTION
**Pull request by mbkuhn**

### CODE REVIEW

File: unit_tests/turbulence/test_turbulence_LES.cpp [hunk 1]
```diff
@@ -0,0 +1,272 @@
...
+
+    // Create turbulence model
+    sim().create_turbulence_model();
+    // Get turbulence model
+    auto& tmodel = sim().turbulence_model();
+
+    // Get coefficients
+    auto model_dict = tmodel.model_coeffs();
+
+    for (const std::pair<const std::string, const amrex::Real> n : model_dict) {
```
Comment (gdeskos) [inline unit_tests/turbulence/test_turbulence_LES.cpp:130]: Not sure why there are two constants

File: unit_tests/turbulence/test_turbulence_LES.cpp [hunk 2]
```diff
@@ -0,0 +1,272 @@
...
+        // Only a single model parameter, Cs
+        EXPECT_EQ(n.first, "Cs");
+        EXPECT_EQ(n.second, Cs);
+    }
+
+    // Constants for fields
+    const amrex::Real srate = 0.5;
```
Comment (gdeskos) [inline unit_tests/turbulence/test_turbulence_LES.cpp:137]: Why is the strain rate constant? Does that mean we only test constant gradients?There is nothing wrong with it, I just wonder if we overlook things by doing that.

File: unit_tests/turbulence/test_turbulence_LES.cpp (hunk 1 above)
Comment (mbkuhn) [reply unit_tests/turbulence/test_turbulence_LES.cpp:130]: there's only one constant. n.first is the name of the constant, and n.second is the value of the constant.

File: unit_tests/turbulence/test_turbulence_LES.cpp (hunk 2 above)
Comment (mbkuhn) [reply unit_tests/turbulence/test_turbulence_LES.cpp:137]: yes, and I'm not exactly testing all the gradients within the strainrate. I can make it more complicated, but I wasn't sure if it was necessary to test more of fvm/strainrate.H. The smagorinsky functionality just calls the strainrate function, and that's how the gradients end up in the calculation.

**Review by gdeskos — APPROVED** [review]

**Pull request merged**